underactuated.html

```diff
@@ -739,37 +748,43 @@
+wheeled robot whose configuration is described by its Cartesian position $x,y$
 and its orientation, $\theta$, so ${\bf q} = \begin{bmatrix} x, y, \theta
-\end{bmatrix}^T$.  The system is subject to differential constraints that
-prevent side-slip, e.g., \begin{gather*} \dot{x} = v \cos\theta \\ \dot{y} = v
-\sin\theta \\ v = \sqrt{\dot{x}^2 + \dot{y}^2} \end{gather*} These constraints
-cannot be integrated into constraint on position--the car can get to any
-configuration $(x,y,\theta)$, it just can't move directly sideways--so they are
-nonholonomic constraints. </div>
+\end{bmatrix}^T$.  The system is subject to a differential constraint that
+prevents side-slip, \begin{gather*} \dot{x} = v \cos\theta \\ \dot{y} = v
+\sin\theta \\ v = \sqrt{\dot{x}^2 + \dot{y}^2} \end{gather*} or equivalently,
+\[\dot{y} \cos \theta - \dot x \sin \theta = 0.\] This constraint
+cannot be integrated into a constraint on configuration&mdash;the car can get to any
+configuration $(x,y,\theta)$, it just can't move directly sideways&mdash;so this is a
+nonholonomic constraint. </div>
 
 Contrast the wheeled robot example with a robot on train tracks.  The train tracks
-would represent a holonomic constraint: if the system was written down in the
-$(x,y,\theta)$ coordinates then the differential track constraints could be
-integrated resulting in a description of the system in a reduced coordinate
-system--train tracks are holonomic constraints. </p>
+correspond to a holonomic constraint: the track constraint can be written directly in
+terms of the configuration ${\bf q}$ of the system, without using the velocity ${\bf \dot{q}}$.
+Even though the track constraint could also be written as a differential constraint
+on the velocity, it would be possible to integrate this constraint to obtain
+a constraint on configuration. The track restrains the possible configurations of the system.
 
-<p> A nonholomic constraint like the no-side-slip constraint on the wheeled
+<p> A nonholonomic constraint like the no-side-slip constraint on the wheeled
 vehicle certainly results in an underactuated system.  The converse is not
-necessarily true--the double pendulum system which is missing an actuator is
-underactauted but would not typically be called a nonholonmic system.  However,
+necessarily true&mdash;the double pendulum system which is missing an actuator is
+underactuated but would not typically be called a nonholonomic system.  However,
 while the definition of a nonholonomic constraint is clear, in my view the
 definition used in the literature for a "nonholonomic system" is more
 ambiguous&dagger;<sidenote>&dagger; Why, for instance, are the Lagrangian
-equations of motion for the double pendulum--clearly non-integrable-- not
+equations of motion for the double pendulum&mdash;clearly non-integrable&mdash;not
 considered nonholonomic constraints?</sidenote>, and we'll generally avoid using
 the term. </p>
```

Comment:
My interpretation: Lagrangian equations of motion are a constraint of the form phi(q, qd, qdd, t) = 0 and cannot be written in the form phi(q, qd, t) = 0 required for a nonholonomic constraint.